js/screens/Profile/Profile.js

```diff
@@ -1,12 +1,77 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Text, View } from "react-native";
+import {
+  Text,
+  View,
+  ScrollView,
+  Image,
+  TouchableOpacity,
+  StyleSheet,
+  ImageBackground
+} from "react-native";
+import SetAReminder from "../../components/SetAReminder/";
+import styles from "./styles";
 
-const Profile = () => {
+const Profile = ({ navigation }) => {
   return (
-    <View>
-      <Text>Profile</Text>
-    </View>
+    <ImageBackground          
+      source={require("../../assets/Background/background.png")}
+      style={{ width: "100%", height: "100%" }}
+    >
+      <ScrollView>
+        <View style={styles.profileContainer}>
+          <Image
+            style={{ width: 120, height: 120 }}
+            source={{ uri: "https://speakbox.ca/images/logo.png" }}
+            // source={require("../../assets/Avatars/beachBear.svg")}
```

Comment:
Remove comments.

Reply:
the comment is the correct image but it is an svg and we have yet found a way to make it work.